docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js

```diff
@@ -52,6 +56,33 @@ const BpkMarkdownRenderer = (props: Props) => {
 
   const renderers = {};
 
+  renderers.paragraph = Paragraph;
+  renderers.link = BpkLink;
+  renderers.blockquote = BpkBlockquote;
+  renderers.tableRow = BpkTableRow;
+  renderers.tableHead = BpkTableHead;
+  renderers.tableBody = BpkTableBody;
+
+  renderers.tableCell = tableCellProps => {
+    const { isHeader, children, ...tableCellRest } = tableCellProps;
```

Comment:
If we don't explicity apply `children` then it needs to be spread
```suggestion
    const { isHeader, ...tableCellRest } = tableCellProps;
```

Reply:
good catch, thanks. Updating now.

Reply:
Done